### PR TITLE
Fixes #16973 - Handle method refactor in clamp 1.10

### PR DESCRIPTION
**Pull request by daviddavis**
More info: https://github.com/mdub/clamp/commit/3d3788a799b88ab0433e4a7ff8fd131144dcfec5

### CODE REVIEW

File: lib/hammer_cli/abstract.rb [hunk 1]
```diff
@@ -137,7 +137,7 @@ def self.build_options(builder_params={})
 
         declared_options << option
         block ||= option.default_conversion_block
-        define_accessors_for(option, &block)
+        declare_attribute(option, &block)
```
Comment (mbacovsky) [inline lib/hammer_cli/abstract.rb]: @daviddavis it won't work with clamp < 1.1.0.

**Comment by mbacovsky** [comment]
Renaming the methods in Hammer would lock us to clamp 1.1.0. We need to support < 1.1.0 too.
I also checked that 1.1.0 also drops ruby 2.0.0 support.So I'd prefer to lock clamp to < 1.1.0 for now.

**Comment by daviddavis** [comment]
@mbacovsky updated

**Comment by mbacovsky** [comment]
:+1: Works well now, thanks, @daviddavis !

**Comment by mbacovsky** [comment]
@daviddavis could you please change the commit message to match the change?

**Comment by mbacovsky** [comment]
@daviddavis thanks! Merging...

**Pull request merged**